src/main/menu/menu.ts

```diff
@@ -9,13 +9,11 @@ import { appName, isMac, isWindows, docsUrl, supportUrl, productName } from "../
 import logger from "../logger";
 import { exitApp } from "../exit-app";
 import { broadcastMessage } from "../../common/ipc";
-import * as packageJson from "../../../package.json";
+import packageJson from "../../../package.json";
 import { preferencesURL, extensionsURL, addClusterURL, catalogURL, welcomeURL } from "../../common/routes";
 import { checkForUpdates, isAutoUpdateEnabled } from "../app-updater";
 import type { MenuRegistration } from "./menu-registration";
 
-export type MenuTopId = "mac" | "file" | "edit" | "view" | "help";
```

Comment:
Not used anywhere.

Reply:
Well, it was used before. Dunno who was refactored menu that it now has some leftovers. 
Anyway, this is out of scope.

Reply:
Nope. This is just like removing a comment since it's not used / touch any other code.

Reply:
Disagree, an upgrade PR like this should be only doing the minimal work to get it working, this should be a separate PR.

Reply:
Okay, I'll remove it, but just want to say.. With that frenetic attitude in PR's you probably gonna work alone on this project ~(maybe this is your goal)~.. Just see how other projects has quick PR reviews without PITA..